kuttl-tests/suites/failure-recovery/node-failure/params.yaml.template

```diff
@@ -1,19 +1,20 @@
 OVERRIDE_CLUSTER_NAME: "Cluster"
-JMX_LOCAL_ONLY: true
+JMX_LOCAL_ONLY: false
 NODE_DISK_SIZE_GIB: 1
 NODE_CPU_MC: "1000"
 NODE_CPU_LIMIT_MC: "1200"
 NODE_MEM_MIB: "1024"
 NODE_MEM_LIMIT_MIB: "1384"
 PROMETHEUS_EXPORTER_ENABLED: "false"
 NODE_COUNT: 2
-NODE_READINESS_PROBE_INITIAL_DELAY_S: 30
-NODE_LIVENESS_PROBE_INITIAL_DELAY_S: 60
+NODE_READINESS_PROBE_INITIAL_DELAY_S: 60
+NODE_READINESS_PROBE_PERIOD_S: 10
+NODE_READINESS_PROBE_FAILURE_THRESHOLD: 8
+NODE_LIVENESS_PROBE_INITIAL_DELAY_S: 140
```

Comment:
The replace bootstrap process can take a bit - there is an initial ring join delay that's 30 seconds by default, so we want to give it enough time to actually replace the node